recipes/fastkml/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "fastkml" %}
+{% set version = "0.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 86d0ee3b108a5508ab5b9ef1b7063fa6afd0115849bbb3f2d6c1e06cda5de279
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - pygeoif
+    - python
+    - python-dateutil
+
+test:
+  imports:
+    - fastkml
+  requires:
+    - pytest
```

Comment:
as you are not running the tests, could you please remove the 
```
  requires:
    - pytest
```

Reply:
@marcelotrevisani removing test caused build to fail

Reply:
The failure is because `setuptools` is missing from the `run` dependencies

Reply:
Thank you @chrisburr